orcid-core/src/main/java/org/orcid/core/manager/v3/validator/ActivityValidator.java

```diff
@@ -392,7 +407,7 @@ private void validateDisambiguatedOrg(MultipleOrganizationHolder organizationHol
         }
     }
 
-    public void validateAffiliation(Affiliation affiliation, Source activeSource, boolean createFlag, boolean isApiRequest, Visibility originalVisibility) {
+    public void validateAffiliation(Affiliation affiliation, Source activeSource, boolean createFlag, boolean isApiRequest, Visibility originalVisibility, AffiliationType type) {
```

Comment:
@Camelia-Orcid please remove the extra param and use `instanceof` to verify the `affiliation` type